script/cache_critical_dns

```diff
@@ -267,12 +268,13 @@ ensure
   client.close if client
 end
 
-def postgres_healthcheck(host:, user:, password:, dbname:)
+def postgres_healthcheck(host:, user:, password:, dbname:, port:)
```

Comment:
The `port` argument for `redis_healthcheck` defaults to `DEFAULT_REDIS_PORT`.

We should either define a default here as well or remove the default value on `redis_healthcheck`.  I'm not that fussed which.

Reply:
It's being overly cautious, but I'll set the default at the arg level here too.